arches/app/templates/base-manager.htm

```diff
@@ -244,6 +245,25 @@ <h5>{% trans "for more documentation, visit" %} <a href="https://arches.readthed
                                         </li>
                                         {% endif %}
 
+                                        
+                                        <li>
+                                            <a href="{% url 'search_home' %}" class="ep-tools ep-tools-right" data-bind="click:function () { navigate('{% url 'search_home' %}') }">
+                                                <div class="" data-placement="bottom" data-toggle="" data-original-title='{% trans "Search" %}'>
+                                                    <i class="ion-search"></i>
```

Comment:
instead of using the `ion-search` icon that we use in the header, lets use `fa fa-search` to achieve consistency amongst the glyphs in the left nav bar.